docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -540,6 +543,261 @@ web-2   gcr.io/google_containers/nginx-slim:0.7
+web-2     1/1       Running   0         18s
+```
+
+Get the Pod's container.
+
+```shell{% raw %}
+get po web-2 --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'
+gcr.io/google_containers/nginx-slim:0.8
+{% endraw %}
+```
+
+Notice that, even though the update strategy is `RollingUpdate` the StatefulSet 
+controller restored the Pod with its original container. This is because the 
+ordinal of the Pod is less than the `partition` specified by the 
+`updateStrategy`.
+
+#### Rolling Out a Canary
+You can roll out a canary to test a modification by decrementing the `partition` 
+you specified [above](#staging-an-update).
+
+Patch the StatefulSet to decrement the partition.
+
+```shell
+kubectl patch statefulset web -p '{"spec":{"updateStrategy":{"type":"RollingUpdate","rollingUpdate":{"partition":2}}}}'
+statefulset "web" patched
+```
+
+Wait for `web-2` to be Running and Ready.
+
+```shell
+kubectl get po -lapp=nginx -w
+NAME      READY     STATUS              RESTARTS   AGE
+web-0     1/1       Running             0          4m
+web-1     1/1       Running             0          4m
+web-2     0/1       ContainerCreating   0          11s
+web-2     1/1       Running   0         18s
+```
+
+Get the Pod's container.
+
+```shell{% raw %}
+kubectl get po web-2 --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'
+gcr.io/google_containers/nginx-slim:0.7
+{% endraw %}
+```
+
+When you changed the `partition`, the StatefulSet controller automatically 
+updated the `web-2` Pod because the Pod's ordinal was less than or equal to 
+the `partition`.
+
+Delete the `web-1` Pod.
+
+```shell
+kubectl delete po web-1
+pod "web-1" deleted
+```
+
+Wait for the `web-1` Pod to be Running and Ready.
+
+```shell
+kubectl get po -lapp=nginx -w
+NAME      READY     STATUS        RESTARTS   AGE
+web-0     1/1       Running       0          6m
+web-1     0/1       Terminating   0          6m
+web-2     1/1       Running       0          2m
+web-1     0/1       Terminating   0         6m
+web-1     0/1       Terminating   0         6m
+web-1     0/1       Terminating   0         6m
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         18s
+```
+
+Get the `web-1` Pods container.
+
+```shell{% raw %}
+get po web-1 --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'
+gcr.io/google_containers/nginx-slim:0.8
+{% endraw %}
+```
+`web-1` was restored to its original configuration because the Pod's ordinal 
+was less than the partition. When a partition is specified, all Pods with an 
+ordinal that is greater than or equal to the partition will be updated when the 
+StatefulSet's `.spec.template` is updated. If a Pod that has an ordinal less 
+than the partition is deleted or otherwise terminated, it will be restored to 
+its original configuration.
+
+#### Phased Roll Outs
```

Comment:
Mention canary and phased rollout in the concept guide under the description of Partition.